src/main/resources/application.yml

```diff
@@ -27,19 +27,14 @@ spring:
   security:
     oauth2:
       client:
-        provider:
+        registration:
           members:
-            client-id: "${spring.security.oauth2.client.registration.members.client-id}"
-            authorization-uri: "https://members.lghs.be/oauth/authorize"
-            token-uri: "https://members.lghs.be/oauth/token"
-            user-info-uri: "https://members.lghs.be/api/me"
+            client-id: "accounting"
+            client-secret: "f539b20e-048a-4d81-90bf-238477fb3ad6"
```

Comment:
Please never include secrets, even fake ones in configs. You never know who will copy paste and it makes debugging harder.
If you need those secrets, drop an application.yml in your work folder and gitignore it (I personally always run project with a CWD outside the git repository to avoid committing secrets by mistake).

Reply:
I'm not fan of having additional configuration to do before launching a development environment.  There is a lot of step before being able to launch the app and this can only introduce error between dev-env/prod or between two developer.
The other MR aims to have a init script, so that you can launch the init script, then docker-compose up and the dev-env is running without any other step (maybe add a line in the host file because I'm not fan of having root privileges to the scripts).
That being said, I understand the issue with having them in the app config file.  I replaced them with env variables and defined them directly in the docker-compose.yaml file